app/src/main/java/com/triadss/doctrack2/activity/patient/fragment/PatientAppointmentAdapter.java

```diff
@@ -0,0 +1,82 @@
+package com.triadss.doctrack2.activity.patient.fragment;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.triadss.doctrack2.R;
+
+import java.util.ArrayList;
+
+// Extends the Adapter class to RecyclerView.Adapter
+// and implement the unimplemented methods
+public class PatientAppointmentAdapter extends RecyclerView.Adapter<PatientAppointmentAdapter.ViewHolder> {
+    ArrayList  courseName;
+    Context context;
+
+    // Constructor for initialization
+    public PatientAppointmentAdapter(Context context,  ArrayList courseName) {
```

Comment:
This is an appointment Pending page, please use appropriate variable names.

Reply:
Changed {courseName} to {Pending}